pyface/data_view/i_data_view_widget.py

```diff
@@ -32,6 +33,12 @@ class IDataViewWidget(IWidget):
     #: Whether or not the column headers are visible.
     header_visible = Bool(True)
 
+    #: The global drop handlers for the data view.  These are intended to
```

Comment:
I'm not sure what you mean by "global" here and what the difference between a global vs local drop handler would be. Is it "global" because it can't handle local drops i.e. drops in a specific row/column/item?

Reply:
Yes.

If needed it _can_ handle row/column/item-specific drops, but you need to start writing toolkit-specific code to dig that information out of the toolkit event and widget objects.